api-reference/v1.0/resources/groupsetting.md

```diff
@@ -0,0 +1,59 @@
+# groupSetting resource type
+
+Group settings can be created based on the available [groupSettingTemplates](groupSettingTemplate.md), and changed from their preset defaults. These settings can govern entity or feature behaviors, both at a tenant-wide level or at a specific entity level. When the same setting is defined at both the tenant-wide and specific entity level, the specific entity level setting may opt-out from the tenant-wide setting.  For example, the tenant-wide setting may allow guests to be invited by existing members of groups, but a specific group setting may opt-out and not allow guests to be invited by members of the group. Currently system defined settings are only govern Office groups behavior.
```

Comment:
Would change the last sentence to:

Groups settings only govern Office groups behavior.